Common/Vulkan/VulkanLoader.cpp

```diff
@@ -239,6 +246,16 @@ void VulkanSetAvailable(bool available) {
 bool VulkanMayBeAvailable() {
 	if (g_vulkanAvailabilityChecked)
 		return g_vulkanMayBeAvailable;
+
+	std::string name = System_GetProperty(SYSPROP_NAME);
+	for (size_t i = 0; i < ARRAY_SIZE(device_name_blacklist); i++) {
```

Comment:
Range based for loop would be a bit cleaner here, I think. Yay for not using macros.